packages/contracts/deploy/00_info/01_info.ts

```diff
@@ -52,6 +53,10 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
       await deployer.getBalance()
     )} native tokens.`
   );
+
+  console.log(
+    `Chosen PluginRepo ENS subdomain name: '${PLUGIN_REPO_ENS_SUBDOMAIN_NAME}'`
+  );
```

Comment:
The ENS name is already being printed in [another step], is this console still needed for making the scripts work? or do we want to keep it for debugging purposes?

Reply:
I thought it's useful information that we want to have in the very beginning.